mitxpro/settings.py

```diff
@@ -599,6 +600,8 @@
 )
 CELERY_TASK_ALWAYS_EAGER = get_bool("CELERY_TASK_ALWAYS_EAGER", False, dev_only=True)
 CELERY_TASK_EAGER_PROPAGATES = get_bool("CELERY_TASK_EAGER_PROPAGATES", True)
+CRON_COURSE_CERTIFICATES_HOURS = get_string("CRON_COURSE_CERTIFICATES_HOURS", 0)
```

Comment:
Staying true to the `get_string` ideology, please update the default to `'0'`.

Reply:
I will go with early approach then.